docs/book/src/cronjob-tutorial/testdata/generate_cronjob.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+#  Copyright 2020 The Kubernetes Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+#
+#  This script is a helper which has just the commands
+#  to generate the conjob tutorial to let us know update manually the testdata dir
+#
+
+set -o errexit
+set -o pipefail
+
+# Turn colors in this script off by setting the NO_COLOR variable in your
+# environment to any value:
+#
+# $ NO_COLOR=1 test.sh
+NO_COLOR=${NO_COLOR:-""}
+if [ -z "$NO_COLOR" ]; then
+  header=$'\e[1;33m'
+  reset=$'\e[0m'
+else
+  header=''
+  reset=''
+fi
+
+build_kb() {
+    go build -o ./bin/kubebuilder sigs.k8s.io/kubebuilder/cmd
+}
+
+function header_text {
+  echo "$header$*$reset"
+}
+
+function gen_cronjob_tutorial {
+  header_text "removing project ..."
+  rm -rf project
+  header_text "starting to generate the cronjob ..."
+  mkdir project
+  cd project
+  header_text "creating tutorial.kubebuilder.io base  ..."
+  kubebuilder init --plugins=go/v3-alpha --domain=tutorial.kubebuilder.io --project-version=3-alpha --repo=tutorial.kubebuilder.io/project --license apache2 --owner "The Kubernetes authors"
```

Comment:
I'm using the latest version of kubebuilder (top of master) and it looks like the `--plugins` flag still doesn't exist here--is that expected / will this come later?

Reply:
It worked when the PR was made. However, now shows that we need to pass the full name of the plugin. I will fix it and also add. the scripts to the Travis. Then we can ensure that it works always. 👍 Tks

Reply:
Hi @gabbifish, 

These scripts are just to allow us re-gen the project inside off this dir and diff with the master to know what requires updates. the plugins only exist on the master branch and by merging this PR we are updating the tutorial with the latest version available which is v3-alpha plugin on the master. It will just be available to production after the next release.

Reply:
IMO what we need to do here is to ensure that we are updating the projects and not losing any text/info. I will mark it with: 

WIP

Because I would like to recheck it too.

Reply:
did you run make install before the script? I will add it on the script for we ensure that it will get the latest bin.